### PR TITLE
feat(LH-72231): SEC Onboarding Resource

### PR DESCRIPTION
**Pull request by weilueluo**
https://jira-eng-rtp3.cisco.com/jira/browse/LH-72231

### Description
- SEC Onboarding resource (create only)
- Onboarding SEC complete example
### Files to read
- client/connector/sec/read_by_name.go
- client/connector/sec/readall.go
- client/connector/sec/seconboarding/create.go

you may also run the example in
- provider/examples/resources/sec
I added the example here because cdo-automation does not have one

### CODE REVIEW

**Comment by hazital** [comment]
can you pls change the base branch to `LH-72231-sec-and-sec-onboarding-resource` so it will be easier to review?

File: provider/examples/resources/sec/sec/main.tf [hunk 1]
```diff
@@ -0,0 +1,29 @@
+data "aws_route53_zone" "selected" {
```
Comment (siddhuwarrier) [inline provider/examples/resources/sec/sec/main.tf:1]: The example structure seems really off here. Can you put the sec directory inside modules?

**Review by siddhuwarrier — APPROVED** [review]
Just one comment, feel free to ignore. My suggestion is that the sec module in the examples be moved in to modules/sec -- otherwise it's a bit confusing to read.

File: provider/examples/resources/sec/sec/main.tf (hunk 1 above)
Comment (weilueluo) [reply provider/examples/resources/sec/sec/main.tf:1]: I will take a note, to fix it properly will need to fix other examples as well, so I will leave it for now and fix them all together when we want to provide complete examples.

**Pull request merged**